_plugins/docs_generator.rb

```diff
@@ -25,7 +25,7 @@ def generate(site)
     site.config['docs_repos'].each do |repo_name, repo_options|
       next unless all_repos.key? repo_name
 
-      repo_path = Pathname.new(File.join("_remotes", repo_name))
+      repo_path = Pathname.new(File.join('_remotes', repo_name))
```

Comment:
I am not sure what style is being used for Ruby but this change looks unrelated and the file mixes both single and double quotes already.

Reply:
It is unrelated, but my OCD triggered and I had to make consistent at least within the same function i.e. do not use double quotes unless interpolating or printing :).